var/spack/repos/builtin/packages/py-importlib-resources/package.py

```diff
@@ -20,6 +24,7 @@ class PyImportlibResources(PythonPackage):
     depends_on('py-setuptools', type='build')
     depends_on('py-setuptools-scm@3.4.1:+toml', when='@5:', type='build')
     depends_on('py-zipp@0.4:', when='@5:', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('py-zipp@0.4:', when='@5.0:5.1', type=('build', 'run'))
```
Otherwise it will get added even when Python 3.10 is used